stix_shifter_modules/nozomi/stix_translation/transformers.py

```diff
@@ -0,0 +1,114 @@
+import re
+from stix_shifter_utils.utils import logger
+from datetime import datetime, timezone
+from stix_shifter_utils.stix_translation.src.utils.transformers import ValueTransformer
+
+LOGGER = logger.set_logger(__name__)
+connector = __name__.split('.')[1]
+
+
+class ChainNameValue(ValueTransformer):
+    """A value transformer to add default 'kill_chain_name' as 'mitre-attack' with obj"""
```

Comment:
can you add input/output values in the comment for all the transformers? its easier for maintenance in future.

Reply:
Added the comment for input/output values.